src/parcels/mount-parcel.js

```diff
@@ -90,19 +90,21 @@ export function mountParcel(config, customProps) {
     customProps,
     parentName: toName(owningAppOrParcel),
     unmountThisParcel() {
-      if (parcel.status !== MOUNTED) {
-        throw Error(
-          formatErrorMessage(
-            6,
-            __DEV__ &&
-              `Cannot unmount parcel '${name}' -- it is in a ${parcel.status} status`,
-            name,
-            parcel.status
-          )
-        );
-      }
-
-      return toUnmountPromise(parcel, true)
+      return mountPromise
```

Comment:
It's conceivable that someone could be relying on `unmountThisParcel()` to throw a synchronous error if the parcel is in the wrong status, which is now changed. However, I doubt that that's the case and am thinking this is probably a minor/patch instead of a breaking change.

Thoughts?

Reply:
Yeah I agree. It's probably not a concern if we want to be uber careful we could do a breaking change but I see this as fixing a bug. If someone was relying on bug behavior I expect it would've come up before now.